components/Layout/DisplayError.tsx

```diff
@@ -0,0 +1,63 @@
+import styled from 'styled-components'
+import React from 'react'
+import PropTypes from 'prop-types'
```

Comment:
Remove this import. If you have added it to your package.json, also remove it from there